packages/server/src/sync/schema.ts

```diff
@@ -12,11 +13,10 @@ export const makeSyncRequestSchema = (channels: Channel[]) => {
     channelsSchema[channel.meta.name] = channel.meta.schema.optional()
   }
 
-  return Joi.object({
-    channels: Joi.object(channelsSchema).allow(null),
-    webhooks: Joi.array().items(SyncWebhookSchema).allow(null),
-    id: Joi.string().guid().allow(null),
-    token: Joi.string().max(125).allow(null),
-    name: Joi.string().max(50).allow(null)
+  return ReqSchema({
+    body: {
+      channels: Joi.object(channelsSchema).allow(null),
+      webhooks: Joi.array().items(SyncWebhookSchema).allow(null)
+    }
   }).options({ stripUnknown: true })
```

Comment:
I think we can now remove this black magic and force users to send proper data. What do you think?

Reply:
We can't remove that unless we do a migration on botpress' side. What we can do is disable this when messaging is not spinned

Reply:
Well plan for future us ;)